### PR TITLE
Fix author link

### PR DESCRIPTION
**Pull request by ardalis**
Added link to author

### CODE REVIEW

File: aspnet/fundamentals/application-insights.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Application Insights
 ====================
 
-By `Steve Smith`
+By `Steve Smith <http://blog.falafel.com/author/steve-smith/>`
```
Comment (danroth27) [inline aspnet/fundamentals/application-insights.rst]: You don't need to inline the URL - just add the missing underscore to pick up the link from authors.txt:

```
`Steve Smith`_
```

Comment (ardalis) [inline aspnet/fundamentals/application-insights.rst]: Oh right, thanks. Forgot that technique.

**Comment by ardalis** [comment]
@danroth27 should be correct now

**Comment by danroth27** [comment]
:shipit:

**Pull request merged**